install_config/topics/configuring_rhv_objects.adoc

```diff
@@ -1,66 +1,410 @@
+= Configuring Red Hat Virtualization
+
+To integrate {product-title} with Red Hat Virtualization, perform the following actions as part of your xref:../install/host_preparation.adoc#install-config-install-host-preparation[host preparation].
+
+== Creating the bastion virtual machine
+
+The bastion virtual machine is used to install {product-title} on Red Hat Virtualization.
+
+.Procedure
+
+. Log in to the Manager machine using SSH.
+. Create a temporary bastion installation directory, for example, *_/bastion_installation_*, for the installation files.
+. Create an encrypted file, *_/bastion_installation/secure_vars.yaml_*:
++
+[options="nowrap" subs="+quotes,verbatim"]
+----
+# ansible-vault create secure_vars.yaml
+----
+
+. Add the following variables to *_secure_vars.yaml_*:
++
+[source,yml]
+----
+engine_password: <Manager_password>
+bastion_root_password: <bastion_root_password>
+rhsub_user: <Red_Hat_Subscription_Manager_username>
+rhsub_pass: <Red_Hat_Subscription_Manager_password>
+rhsub_pool: <Red_Hat_Subscription_Manager_pool_id>
+root_password: <OpenShift_node_root_password>
+----
+
+. Save the file and record the vault password.
+
+. Create the installation playbook, *_/bastion_installation/create-bastion-machine-playbook.yaml_*, and update its parameters:
++
+[source,yml]
+----
+---
+- name: Create a bastion machine
+  hosts: localhost
+  connection: local
+  gather_facts: false
+
+  vars:
+    engine_url: https://_Manager_FQDN_/ovirt-engine/api
+    engine_user: <admin@internal>
+    engine_password: "{{ engine_password }}"
+    engine_cafile: /etc/pki/ovirt-engine/ca.pem
+
+    qcow_url: <1>
+    template_cluster: Default
+    template_name: rhelguest7
+    template_memory: 4GiB
+    template_cpu: 2
+    wait_for_ip: true
+    debug_vm_create: false
 
-To integrate {product-title} with Red Hat Virtualization, take the following
-actions as part of your
-xref:../install/host_preparation.adoc#install-config-install-host-preparation[host preparation].
+    vms:
+    - name: rhel-bastion
+      cluster: Default
+      profile:
+        cores: 2
+        template: "{{ template_name }}"
+        root_password: "{{ root_password }}"
+        ssh_key: "{{ lookup('file', '/root/.ssh/id_rsa_ssh_ocpadmin.pub') }}"
+        state: running
+      cloud_init:
+        custom_script: |
+          rh_subscription:
+            username: "{{ rhsub_user }}"
+            password: "{{ rhsub_pass }}"
+            auto-attach: true
+            disable-repo: ['*']
+            # 'rhel-7-server-rhv-4.2-manager-rpms' supports RHV 4.2 and 4.3
+            enable-repo: ['rhel-7-server-rpms', 'rhel-7-server-extras-rpms', 'rhel-7-server-ansible-2.7-rpms', 'rhel-7-server-ose-3.11-rpms', 'rhel-7-server-supplementary-rpms', 'rhel-7-server-rhv-4.2-manager-rpms']
+          packages:
+            - ansible
+            - ovirt-ansible-roles
+            - openshift-ansible
+            - python-ovirt-engine-sdk4
+  tasks:
+    - user
```

Comment:
@rgolangh @apinnick This throws a syntax error since we're missing a colon. And since we  need to edit this, we might as well give this task a `name:` directive.

Reply:
Speaking of which, why do we even need this task?

Reply:
Do you want me to delete the task?

Reply:
I am not saying that since I am really not sure if we need it or not. It seems unnecessary to me, but there's probably a good reason why it's there and I just cannot see it.

Reply:
OK